### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone or `ldd`. You don't need to set `AFL_SKIP_CPUFREQ`

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+image: ghcr.io/capuanob/pdf-tools-fuzz:latest
```
Comment (xansec) [inline Mayhemfile:1]: Tests -> testsuite, and omit image name.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**